package.json

```diff
@@ -78,6 +79,9 @@
     "@discordjs/opus": {
       "optional": true
     },
+    "@stablelib/xchacha20poly1305": {
+      "optional": true
+    },
```

Comment:
nit: `@stablelib/xchacha20poly1305` isn't a peer dependency (not decided on whether to force users to pick between stablelib and https://github.com/projectdysnomia/libsodium yet – that variant is still in play)